pci/graphic_drivers/xf86-video-intel/MHWDCONFIG

```diff
@@ -15,35 +15,24 @@ BLACKLISTEDDEVICEIDS="0be1 8108"
 MHWDCONFLICTS="video-hybrid-intel-nvidia-bumblebee video-hybrid-intel-nouveau-bumblebee"
 
 # Dependencies
-DEPENDS="xf86-video-intel"
-
+DEPENDS="xf86-video-intel vulkan-intel libva-mesa-driver"
+DEPENDS_64="lib32-vulkan-intel"
 
```

Comment:
It's at the same level as radeon-vulkan(radv) wich means "Dota2" & "The Talos Principle" works (with bad FPS, buuuut thats the same for radv). Also right now vulkan needs to be enabled by the users in these 2 games.